src/pages/gl-integration-logic/Forms/IntegrationLogicDetails.js

```diff
@@ -0,0 +1,124 @@
+import { useContext, useEffect, useState } from 'react'
+import toast from 'react-hot-toast'
+import Table from 'src/components/Shared/Table'
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useWindow } from 'src/windows'
+import { ControlContext } from 'src/providers/ControlContext'
+import { GeneralLedgerRepository } from 'src/repositories/GeneralLedgerRepository'
+import IntegrationLogicDetailsForm from './IntegrationLogicDetailsForm'
+
+const IntegrationLogicDetails = ({  labels, maxAccess, store }) => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const [gridData, setGridData] = useState([])
+  const { stack } = useWindow()
+  const { recordId } = store
+  const editMode = !!store.recordId
```

Comment:
since editMode used only for IntegrationLogicDetailsForm  props remove it and add const editMode inside file itself not here